apps/home/routes.py

```diff
@@ -393,6 +393,157 @@ def view_labs(lab_id=None):
     return render_template("pages/labs_view.html", labs=labs, lab_categories=lab_categories, running_labs=running_labs, segment="/labs/view")
 
 
+@blueprint.route('/groups/', defaults={'group_id': None})
+@blueprint.route('/groups/<int:group_id>')
+@login_required
+def groups(group_id):
```

Comment:
this function is not making much sense. Can you please double check if this was left behind from previous commits?